rusk-wallet/src/bin/command.rs

```diff
@@ -410,14 +420,30 @@ pub(crate) enum Command {
         amt: Dusk,
 
         /// Max amount of gas for this transaction
-        #[clap(short = 'l', long, default_value_t = DEFAULT_STAKE_GAS_LIMIT)]
+        #[clap(short = 'l', long, default_value_t = DEFAULT_LIMIT_TRANSFER)]
```

Comment:
Conversions are managed by contract-calls too, so the `DEFAULT_LIMIT_CALL` should be used here